spec/evaluators/fictium/postman_evaluator_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Fictium::PostmanEvaluator do
+  subject(:evaluator) { described_class.new(target) }
+
+  let(:target) { Fictium::Document.new }
+
+  describe '#evaluate' do
+    before do
+      evaluator.evaluate do
+        pre_request script: 'console.log("A");'
+        test script: 'console.log("TEST");'
+        variable name: 'x'
+        auth apikey: { key: 'token' }
+      end
+    end
+
+    it 'evaluates postman properties' do
```

Comment:
:thinking: this example is blank, it's ok?

Reply:
No, it isn't, strange it's not a COP...